src/datadog/trace_segment.cpp

```diff
@@ -389,37 +414,32 @@ void TraceSegment::write_sampling_delegation_response(DictWriter& writer) {
     j["mechanism"] = *sampling_decision_->mechanism;
   }
 
-  if (expecting_delegated_sampling_decision_) {
-    expecting_delegated_sampling_decision_ = false;
-    trace_tags_.emplace_back(tags::internal::sampling_decider, "1");
-  }
-
-  writer.set(DD_SAMPLING_DELEGATION_RESPONSE_HEADER, j.dump());
-  return;
+  writer.set(sampling_delegation_response_header, j.dump());
```

Comment:
meeting with Damien: Maybe we should decide on a "static" constant naming convention. `DD_CAPS_STUFF` might work, but looks like an env variable. `some_thing` looks like a local. Maybe `k_foo_bar` or similar?

Reply:
We can decide later.